packages/matter.js/src/protocol/ExchangeManager.ts

```diff
@@ -53,6 +56,7 @@ export class ExchangeManager<ContextT> {
     private readonly exchanges = new Map<number, MessageExchange<ContextT>>();
     private readonly protocols = new Map<number, ProtocolHandler<ContextT>>();
     private readonly transportListeners = new Array<Listener>();
+    private readonly sessionsInClosing = new Set<number>();
```

Comment:
Kind of an odd name, maybe "closingSessions" or "sessionsInCloseSession", or do I misunderstand the meaning?

Reply:
I renamed